overloads/vector.h

```diff
@@ -74,5 +108,24 @@ vector<T> exponent(const vector<T> &a) {
     return arr;
 }
 
+// isEquals operator
+template<typename T>
+bool operator == (const vector<T> &a, const vector<T> &b) {
+    int n = a.size();
```

Comment:
supernit: vector.size() returns an unsigned int, but we're typecasting it to int here. If there is a scenario where this typecasting leads to overflow, this condition might fail even though the vectors have the same size. A cleaner way to check would be to just have the size method calls inside the if condition directly.

Reply:
Just converted the n and m to type size_t. Now the size will not overflow as size_t is the base numeric data type used throughout C++.  It is the base class of all float, double, int values.